src/audio/opus/celt_decoder.c

```diff
@@ -194,15 +194,15 @@ int celt_decoder_init(CELTDecoder *st, opus_int32 sampling_rate, int channels)
       return ret;
    st->downsample = resampling_factor(sampling_rate);
    if (st->downsample==0)
-      return OPUS_BAD_ARG;
+      assert(0);
```

Comment:
Why these changes are necessary?

Reply:
It was easier to debug when I knew where the assert was. Otherwise the faulting location was non obvious because it was the calling function that would assert. If you prefer I can leave this out.

Reply:
I'd like to minimize the changes in libopus itself to facilitate merges. It's true that asserts in our codebase make it easier to debug but that's true for hundreds of similar return codes, and we can't just change the whole libopus.